src/utils/enums.ts

```diff
@@ -0,0 +1,13 @@
+export enum Filters {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
+
+export enum Errors {
```

Comment:
Consider renaming `Errors` enum to a more specific name like `TodoErrors` or `AppErrors` to avoid confusion with other error enums in the future.